src/components/flame-graph/Canvas.js

```diff
@@ -48,6 +50,64 @@ const ROW_HEIGHT = 16;
 const TEXT_OFFSET_START = 3;
 const TEXT_OFFSET_TOP = 11;
 
+/* Return the background and foreground colors used to paint a box in
```

Comment:
This is a small style tweak, but generally we've done the following style for long-form comments:

```
/**
 * Long form comment
 */
```

Also, thanks for writing long form comments!